bigip/resource_bigip_ltm_monitor.go

```diff
@@ -148,10 +170,12 @@ func resourceBigipLtmMonitorRead(d *schema.ResourceData, meta interface{}) error
 			d.Set("manual_resume", m.ManualResume)
 			d.Set("parent", m.ParentMonitor)
 			d.Set("name", name)
+			d.Set("username", m.Username)
+			d.Set("password", d.Get("password").(string))
```

Comment:
> Currently there are no public facing documents detailing the hashing and the associated solution article is very vague to the point of not covering the extent of the change as this encrypted monitor passwords not only FTP monitors.

> SOL16146: FTP monitor passwords are not encrypted
    https://support.f5.com/kb/en-us/solutions/public/16000/100/sol16146.html

> From the notes the BIG-IP system uses internal keys to encrypt the data and will decrypt to send the monitor string to the target device as the implication is this is encrypted in the configuration to avoid easy access to password information.

Sounds like this is a 👎